requirements/requirements.dev.txt

```diff
@@ -2,6 +2,7 @@ black~=22.0
 flake8>=4.0.1
 gitlint-core==0.17.0
 isort>=5.10.1
-mypy>=0.941,<=0.971
+mypy>=1.4.1
```

Comment:
would this not also possibly install 1.5.1 for 3.7? Is that what you want? This just sets a different min version for 3.8 and higher.

Reply:
Hmm.  Not sure ... I just copied it from Studio.

Reply:
I think @joanise was the one who introduced it there?  My guess is that 1.5.1 isn't actually installable on 3.7, and we don't specifically depend on its features, but we'd *like* to have it if possible elsewhere.

Reply:
Exactly, the most recent mypy available for Python 3.7 is 1.4.1.

My commit log message for that change says "fix: make Studio compatible with Pydantic 2 and thus g2p 2", and I'm sure I had a good reason to do it this way, but that was a whole month ago, so I don't remember what it was.